cloudify_agent/installer/config/agent_config.py

```diff
@@ -391,7 +387,7 @@ def _set_package_url(self, runner):
 
         if agent_package_name:
             file_server_url = agent_utils.get_manager_file_server_url(
-                self['rest_host'], self['rest_port']
+                self['rest_host'][0], self['rest_port']
```

Comment:
this (and the one below) is to be replaced with a random.choice probably. The point is, in one attempt of an installation, we will only download the package from one manager, relying on retries to try the other managers

Reply:
When is this going to be replaced?
(and isn't there something in common for this?)